pkg/docker/docker.go

```diff
@@ -199,7 +207,7 @@ func (rco RunContainerOptions) asDockerConfig() dockercontainer.Config {
 		Image:        getImageName(rco.Image),
 		User:         rco.User,
 		Env:          rco.Env,
-		Entrypoint:   dockerstrslice.StrSlice(rco.Entrypoint),
+		Entrypoint:   rco.Entrypoint,
```

Comment:
sorry if this was here before and I missed earlier - just a question / need for confirmation - was there a type change for this field in engine-api?